modules/post/windows/manage/migrate.rb

```diff
@@ -42,7 +42,7 @@ def run
     if datastore['SPAWN']
       print_status("Spawning notepad.exe process to migrate to")
       target_pid = create_temp_proc
-    elsif datastore['PID'] != 0
```

Comment:
Aha! That does make sense, I should've noticed that and mentioned it in #8109. Thanks!